lib/editing/editor_codemirror.dart

```diff
@@ -116,20 +116,39 @@ class CodeMirrorFactory extends EditorFactory {
         );
       }).toList();
 
+      List<HintResult> filterHints = new List.from(hints, growable: true);
+
+      // If we have 2 hint results with the same displaytext
+      // where one is a getter and the other setter,
+      // we merge them as 1 result with classname: "type-property"
+      for (HintResult result in hints) {
+        for (HintResult other in hints) {
+          if (result.displayText  == other.displayText && result.className == "type-getter" && other.className == "type-setter") {
```

Comment:
nit: there are 2 spaces before the `==`

also, can you wrap this line at 80 cols?